scss/main/_item.scss

```diff
@@ -286,83 +244,70 @@
                                             transform: translate(-38%, -50%) skew(16deg);
                                         }
                                     }
-
                                     &:hover {
                                         opacity: 0.5;
                                     }
                                 }
                             }
                         }
                         .add-img-wrapper {
-                                background-color: #666666;
-                                height: 100%;
-                                width: 357px;
-                                display: inline-block;
-                                position: absolute;
-                                overflow: hidden;
-
-                                .en & {
-                                    transform: skew(16deg);
-                                    left: 327px;
-                                }
-                                .he & {
-                                    transform: skew(-16deg);
-                                    right: 327px;
-                                }
-
-                                .picture {
-                                    &--display-more {
-
-                                        .en & {
-                                            transform: translate(-50%, -50%) skew(-16deg);
-                                        }
-                                        .he & {
-                                            transform: translate(-50%, -50%) skew(16deg);
-                                        }
+                            background-color: #666666;
+                            height: 100%;
+                            width: 357px;
+                            display: inline-block;
+                            position: absolute;
+                            overflow: hidden;
+                            .en & {
+                                transform: skew(16deg);
+                                left: 327px;
+                            }
+                            .he & {
+                                transform: skew(-16deg);
+                                right: 327px;
+                            }
+                            .picture {
+                                &--display-more {
+                                    .en & {
+                                        transform: translate(-50%, -50%) skew(-16deg);
                                     }
-
-                                    &:hover {
-                                        opacity: 0.5;
+                                    .he & {
+                                        transform: translate(-50%, -50%) skew(16deg);
                                     }
                                 }
-
+                                &:hover {
+                                    opacity: 0.5;
+                                }
```

Comment:
looks like &:hover moved to different place - need to make sure it doesn't have negative / unrelated / unexpected effects